app/components/GroupForm/GroupForm.php

```
@@ -0,0 +1,110 @@
+            $this['groupForm']->addSubmit('submit', 'Create')
+                ->getControlPrototype()
+                ->addClass('btn btn-success');
+        }
+        $this->template->render();
+    }
+
+    /**
+     * @param Form $form
+     * @param ArrayHash $values
+     */
+    public function formSent(Form $form, ArrayHash $values)
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$form'. (undefined)

(UnusedFormalParameter)